base/linalg/symmetric.jl

```diff
@@ -423,10 +470,8 @@ function expm{T}(A::Hermitian{T})
 end
 
 for (funm, func) in ([:logm,:log], [:sqrtm,:sqrt])
-
     @eval begin
-
-        function ($funm)(A::Symmetric)
+        function ($funm){T<:Real}(A::Symmetric{T})
```

Comment:
No, the issue is that `Symmetric{<:Complex}` may have ill-conditioned eigenvectors `F.vectors`.  Whereas for real-symmetric matrices the eigenvectors are unitary.  The complex analogue is a Hermitian matrix, not complex-symmetric.